src/components/form/Form.jsx

```diff
@@ -0,0 +1,28 @@
+import PropTypes from "prop-types";
+import theme from "@/themes/theme";
+
+const Textarea = ({ className, children, id, name, onSubmit, onReset }) => (
+  <form
+    css={[theme.form.base, className]}
+    id={id}
```

Comment:
Do we need `id`?  Can we not just set `id` to whatever `name` is set to?  Or, alternatively, let's set the default value of `id` to whatever `name` is set to and only override it if `id` is explicitly set.  

That reminds me -- are any of these props required?  If so, should we not indicate that in propTypes?